R/support_panel.R

```diff
@@ -0,0 +1,207 @@
+library(shiny)
```

Comment:
Don't think we need these library calls here... need them listed under imports in the function header.